packages/core/src/browser/tree/tree-model.ts

```diff
@@ -44,6 +44,10 @@ export interface ITreeModel extends ITree, ITreeSelectionService, ITreeExpansion
      */
     openNode(node?: ITreeNode | undefined): void;
     /**
+     * Event for when the user double clicks a node.
```

Comment:
A node can also be opened programmatically or on other triggers.

Reply:
I found it strange to assume that a ndoe would be 'opened'. It is more like a primary action on a node, that could trigger many different things. But since it is in the name I will align the documentation.